src/qtiCommonRenderer/renderers/interactions/TextEntryInteraction.js

```diff
@@ -72,13 +72,23 @@ var showTooltip = function showTooltip($input, theme, message) {
  */
 var render = function render(interaction) {
     var attributes = interaction.getAttributes(),
+        baseType = interaction.getResponseDeclaration().attributes.baseType,
         $input = interaction.getContainer(),
         expectedLength,
         updateMaxCharsTooltip,
         updatePatternMaskTooltip,
         patternMask = interaction.attr('patternMask'),
         maxChars = parseInt(patternMaskHelper.parsePattern(patternMask, 'chars'), 10);
 
+    // Setting up baseType
+    if (baseType) {
+        if (baseType === 'integer') {
+            $input.attr('inputmode', 'numeric');
+        } else if (baseType === 'float') {
+            $input.attr('inputmode', 'decimal');
+        }
+    }
+
```

Comment:
is it expected not to have a value for `inputmode` attribute in case there is no `baseType`?

Reply:
`input` by default is string, so when user click, will appear the default keyboard.